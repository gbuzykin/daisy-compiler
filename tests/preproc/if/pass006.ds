#define ENABLE
#if defined(ENABLE)
#info "case #1"
#endif

#if defined(_NABLE)
#info "case #2"
#endif
