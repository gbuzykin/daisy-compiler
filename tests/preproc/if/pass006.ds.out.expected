./preproc/if/pass006.ds:3:2: info: "case #1"
 3 | #info "case #1"
   |  ^~~~
./preproc/if/pass006.ds: info: warnings 0, errors 0
